### PR TITLE
`EffectEnIceBlock` OK, `object_ice_block` documented (and minor `Obj_Iceblock` documented)

### PR DESCRIPTION
**Pull request by isghj5**
Before opening this PR, ensure the following:
- `./format.sh` was run to apply standard formatting.
- `make` successfully builds a matching ROM.
- No new compiler warnings were introduced during the build process.
    - Can be verified locally by running `tools/warnings_count/check_new_warnings.sh`
- New variables & functions should follow standard naming conventions.
- Comments and variables have correct spelling.
---

https://user-images.githubusercontent.com/42048411/159821871-9dc75964-2f81-4ae9-9a88-f0eebfda4830.mov

I got ahead of myself: this is my first effect so I need verification that I am decomping effects correctly but I got distracted and started documenting actor:`Obj_Iceblock` before I realized how large and complicated of an actor it is, so this includes some basic documentation from that actor, but I don't want to document that actor any further right now since the Effect should be the focus, and it could take me awhile to tackle Obj_Iceblock.

### CODE REVIEW

**Review by AngheloAlf — APPROVED** [review]

File: src/overlays/effects/ovl_Effect_En_Ice_Block/z_eff_en_ice_block.c [hunk 1]
```diff
@@ -1,31 +1,85 @@
...
+        Matrix_InsertTranslation(this->pos.x, this->pos.y, this->pos.z, MTXMODE_NEW);
+        Matrix_Scale(scale, scale, scale, MTXMODE_APPLY);
+        Matrix_InsertMatrix(&globalCtx->billboardMtxF, MTXMODE_APPLY);
+        Matrix_InsertZRotation_s(this->rRot, MTXMODE_APPLY);
+
+        gSPMatrix(POLY_XLU_DISP++, Matrix_NewMtx(gfxCtx), G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
+
+        gSPDisplayList(POLY_XLU_DISP++, &gIceBlockShardEffectDL);
+
+        CLOSE_DISPS(globalCtx->state.gfxCtx);
```
Comment (hensldm) [inline src/overlays/effects/ovl_Effect_En_Ice_Block/z_eff_en_ice_block.c:78]:
```suggestion
        CLOSE_DISPS(gfxCtx);
```
Was looking at the decompme scratch, and this still matches and seems better imo.

**Review by hensldm — APPROVED** [review]
Nice work. Just one minor comment about the draw function in the effect.

File: src/overlays/effects/ovl_Effect_En_Ice_Block/z_eff_en_ice_block.c (hunk 1 above)
Comment (AngheloAlf) [reply src/overlays/effects/ovl_Effect_En_Ice_Block/z_eff_en_ice_block.c:78]: I think I prefer if we used the same parameter as the one used on the  `OPEN_DISP

Comment (isghj5) [reply src/overlays/effects/ovl_Effect_En_Ice_Block/z_eff_en_ice_block.c:78]: Oh cool it does match, yeah I just assumed it didn't, will change.

**Review by hensldm — COMMENTED** [review]
Just noticed this while decomping one of the other effects, but you should be able to remove the `ovl_Effect_En_Ice_Block` section from `undefined_syms.txt`

**Review by Kenix3 — APPROVED** [review]

**Pull request merged**